### ISSUE TITLE
Unable to Place Armor Stands and Cakes

### ISSUE THREAD

**Issue by sRedFlower** [GeyserMC/Geyser#5008]
### Describe the bug

Hello,

I am experiencing an issue where I am unable to place armor stands and cakes in the game. This problem occurs on my server running the latest version of GeyserMC. I have tested it on both the Bedrock and Java editions, and the issue persists.

### To Reproduce

1. Attempt to place an armor stand or cake in the game.
2. Observe that the item does not get placed.

### Expected behaviour

The armor stand or cake should be placed in the game when I attempt to do so.

### Screenshots / Videos

The item does not get placed, and there is no error message or feedback indicating why.

### Server Version and Plugins

This server is running Paper version 1.21-110-master@645a677 (2024-07-27T13:19:54Z) (Implementing API version 1.21-R0.1-SNAPSHOT)
Previous version: 1.21-51-f7124df (MC: 1.21)

### Geyser Version

Geyser version 2.4.2-b660 (git-master-63e60bc) (Java: 1.21、 Bedrock: 1.20.80/1.20.81 - 1.21.20/1.21.21)

### Minecraft: Bedrock Edition Device/Version

Bedrock version: 1.21.21

**Comment by Camotoy** [comment]
Are you saying the issue occurs on both Java and Bedrock?

**Comment by sRedFlower** [comment]
> 您是說 Java 和 Bedrock 都出現這個問題嗎？

No, the issue only occurs on Bedrock Edition (mobile). Everything works fine on Java Editi

**Comment by AJ-Ferguson** [comment]
Is this in creative mode? Try using a single item instead of a stack.

**Comment by sRedFlower** [comment]
> Is this in creative mode? Try using a single item instead of a stack.

Thank you for your suggestion. I tested the issue further and found the following:

- Whether in Creative Mode or Survival Mode, I can place a single cake or armor stand without any issues.
- However, when I try to place a stack of these items, they do not get placed.
- Additionally, after logging out and logging back in, the items that were in the stack disappear from my inventory.

It seems that the issue only occurs when trying to place multiple items at once.

**Comment by Ryuu-Shin** [comment]
here is the video, and other bugs
https://www.mediafire.com/folder/0jy52ofu6yoa8/Bug

**Comment by BenHeroPH** [comment]
this bug still occurs on latest geyser build - Geyser version 2.5.0-b720 (git-master-c240c1c) (Java: 1.21.3, Bedrock: 1.20.80/1.20.81 - 1.21.40 - 1.21.44)

**Comment by oryxel1** [comment]
From: #5612 but posted here so there is at least a video showing the issue.

https://github.com/user-attachments/assets/2c61f219-766a-4c0f-8919-54b0b7020f74
